File: 75-sort-colors/75-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        n = len(nums)
        for i in range(n-1,-1,-1):
            maxIndex = i
            for j in range(i):
                if nums[j] > nums[maxIndex]:
                    maxIndex = j
            nums[i], nums[maxIndex] = nums[maxIndex], nums[i]
        return nums
        

File: 75-sort-colors/75-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int N = nums.length;
        int less = 0, larger = N - 1;
        for(int i = 0; i <= larger ;){
            if(nums[i] == 0){
                int temp = nums[i];
                nums[i] = nums[less];
                nums[less] = temp;
                less++;
                i++;
            }else if(nums[i] == 2){
                int temp = nums[i];
                nums[i] = nums[larger];
                nums[larger] = temp;
                larger--;
            }else{i++;}
        }

        
    }
}